apps/changelog-generator/src/Cli.ts

```diff
@@ -1,8 +1,108 @@
-import * as Command from '@effect/cli/Command';
+import { Command, Options } from '@effect/cli';
+import type { ConfigType, DeploymentConfigV2 } from '@blocksense/config-types';
+import {
+  configDir,
+  readConfig,
+  readEvmDeployment,
+} from '@blocksense/config-types';
+import { Console, Effect } from 'effect';
+import Table from 'cli-table3';
+import type { NetworkName } from '@blocksense/base-utils';
+import { parseNetworkName } from '@blocksense/base-utils';
+import { Terminal } from '@effect/platform';
 
-const command = Command.make('hello');
+const listFeeds = Command.make(
+  'list-feeds',
+  {
+    dir: Options.directory('dir').pipe(Options.withDefault(configDir)),
+  },
+  ({ dir }) =>
+    Effect.gen(function* (_) {
+      const network: NetworkName = yield* _(getNetwork);
+      const config = yield* _(
+        Effect.tryPromise(() => readConfig('feeds_config_v2', dir)),
+      );
+      yield* _(printFeeds(config, network));
+    }),
+);
+
+const command = Command.make('changelog-generator').pipe(
+  Command.withSubcommands([listFeeds]),
+);
 
 export const run = Command.run(command, {
-  name: 'Hello World',
+  name: 'Changelog Generator',
   version: '0.0.0',
 });
+
+const printFeeds = (
+  config: ConfigType<'feeds_config_v2'>,
+  network: NetworkName,
+) =>
+  Effect.gen(function* (_) {
+    const deploymentData = yield* _(getDeploymentData(network));
+    if (!deploymentData) {
+      yield* Console.error(
+        `Deployment data for v2 not found for network: '${network}'`,
+      );
+      return;
+    }
+
+    const table = yield* _(initTable());
+
+    config.feeds.forEach(feed => {
+      const contract =
+        deploymentData?.contracts?.CLAggregatorAdapter[Number(feed.id)];
```

Comment:
```suggestion
        deploymentData?.contracts?.CLAggregatorAdapter[`${feed.id}`];
```

A feed id can be larger than [`Number.MAX_SAFE_INTEGER`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER) (`2**53 – 1`), as it is defined as `u128`, so a conversion to `number` could be a bug. Also the keys are strings, so you can just call `.toString()` on the id, or better yet - interpolate it.